buildrunner/steprunner/tasks/__init__.py

```diff
@@ -6,19 +6,21 @@
 with the terms of the Adobe license agreement accompanying it.
 """
 
+from buildrunner.config.models_step import StepTask
+
 
 class BuildStepRunnerTask:
     """
     Base task class.
     """
 
-    def __init__(self, step_runner, config):
+    def __init__(self, step_runner, step: StepTask):
```

Comment:
Should we also add typing for step_runner?

Reply:
That's harder since it could one of many types/subclasses I believe.